frontend/src/metabase/selectors/app.ts

```diff
@@ -20,6 +20,7 @@ export interface RouterProps {
 }
 
 const PATHS_WITHOUT_NAVBAR = [
+  /^\/setup/,
```

Comment:
navbar and sidebar would appear when the setup endpoint was called, but the user was not finished with the setup process